test/challsrv/README.md

```diff
@@ -0,0 +1,111 @@
+# Challenge Test Server
+
+The `chal-test-srv` package offers a library/command that can be used by test
```

Comment:
It looks like you originally called this chal-test-srv but renamed it to challsrv. Can you talk briefly about why? Especially with these shiny new docs, I worry someone will decide to use this for their live validation. At a minimum we should add "trivially insecure" and "test only" to the docs.

Reply:
> It looks like you originally called this chal-test-srv but renamed it to challsrv. Can you talk briefly about why? 

I renamed it to make the library usage of the package match the Golang package naming conventions.

>  Especially with these shiny new docs, I worry someone will decide to use this for their live validation. At a minimum we should add "trivially insecure" and "test only" to the docs.

:+1: I'm happy to make it clearer this is designed for testing only. We could also make the binary name `chall-test-srv` and leave the package name `challsrv` if you think that would be valuable.

Reply:
What about calling both `challtestsrv`?

Reply:
Sure, that's fine with me :+1:

Reply:
Renamed in https://github.com/letsencrypt/boulder/pull/3689/commits/9e02c2f995c45e5648a501a7470be49f6b6a36e7 I also added a warning to the README about it being insecure test-only code.